server/src/integration/java/org/opentosca/toscana/core/parse/converter/NodeConverterIT.java

```diff
@@ -0,0 +1,72 @@
+package org.opentosca.toscana.core.parse.converter;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javax.xml.namespace.QName;
+
+import org.opentosca.toscana.core.BaseIntegrationTest;
+
+import com.google.common.collect.Sets;
+import org.eclipse.winery.model.tosca.yaml.TNodeTemplate;
+import org.eclipse.winery.yaml.common.Defaults;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static org.opentosca.toscana.core.parse.converter.NodeConverter.TOSCA_PREFIX;
+
+/**
+ Tests whether all natively supported node types of the winery parser are also known to the {@link NodeConverter}.
+ */
+public class NodeConverterIT extends BaseIntegrationTest {
+
+    private static final Logger logger = LoggerFactory.getLogger(NodeConverterIT.class);
+
+    @Test
+    public void supportForAllWineryNodeTypes() throws UnknownNodeTypeException {
+        NodeConverter converter = new NodeConverter();
+        Set<String> wineryNodeTypes = getWineryNodeTypes();
+        for (String nodeType : wineryNodeTypes) {
+            logger.info("Testing conversion of type '{}'", nodeType);
+            TNodeTemplate template = new TNodeTemplate();
+            template.setType(new QName(nodeType));
+            try {
+                converter.convert("name", template);
+                logger.info("Node Type '{}': known", nodeType);
+            } catch (UnsupportedOperationException e) {
+                logger.info("Node Type '{}': known (not yet supported)", nodeType);
+            } finally {
+                System.out.println();
```

Comment:
is this needed?

Reply:
Yes, it increases the test output readability